src/k8s/pkg/k8sd/features/cilium/network.go

```diff
@@ -60,6 +70,16 @@ func ApplyNetwork(ctx context.Context, snap snap.Snap, apiserver types.APIServer
 		}, err
 	}
 
+	ciliumNodePortValues := map[string]any{
+		"enabled": true,
+		// kube-proxy also binds to the same port for health checks so we need to disable it
+		"enableHealthCheck": false,
+	}
+
+	if config.directRoutingDevice != "" {
```

Comment:
nit: Maybe we can also check that these config values are present in the final applied helm `values` in `network_test.go`.